src/parse_type.rs

```diff
@@ -109,6 +110,115 @@ pub(crate) fn consume_generic_params(tokens: &mut TokenIter) -> Option<GenericPa
     })
 }
 
+fn consume_generic_arg(tokens: Vec<TokenTree>) -> GenericArg {
```

Comment:
To follow venial's naming convention, this should be called `parse_generic_arg`.

Reply:
Done.